documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,83 @@
+# Get Bootchart from a Clear Container#
+
+The following steps describe how to get a bootchart from a Clear Containers Container.
+Currently systemd is the default init process in the Clear Containers mini-os
+system. Systemd provides a tool called systemd-bootchart that
```

Comment:
`systemd-bootchart`.  I guess "systemd" could be either the project or the binary so that could also be put in backticks.